homeassistant/components/fibaro/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Instead of calling functions `unknown_failure_1`, let's parametrize them. I think then you can also inline the `_recovery_after_failure_works` again

Reply:
Agree: Status changed and comment added to quality_scale.yaml